pandasai/__init__.py

```diff
@@ -147,6 +150,7 @@ def __init__(
             conversational=conversational,
             verbose=verbose,
             enforce_privacy=enforce_privacy,
+            sample_head=sample_head,
```

Comment:
I wouldn't add it here. This is just to ensure the backwards compatibility. We don't want new features to be supported here, or it will be tough to discontinue this at some point in the future! What do you think?